File: Python/1064.py
a = float(input())
b = float(input())
c = float(input())
d = float(input())
e = float(input())
f = float(input())
media = 0
cont = 0
t = [a,b,c,d,e,f]
for x in t: 
    if x > 0:
        cont = cont + 1
        media = media + (x)
print('{} valores positivos'.format(cont))
print('{:.1f}'.format(media/cont))

File: URI/1064.js
let input = require ('fs').readFileSync('./dev/stdin/file.txt','utf8')
let = lines = input.split('\n')
let contadorDePositos = 0
let total = 0
let valor 
for (let i = 0; i < 6; i++) {
    valor = +lines.shift()
    if (valor>0){
        contadorDePositos ++
        total = total + valor
    }    
}
console.log(`${contadorDePositos} valores positivos`)
console.log((total/contadorDePositos).toFixed(1))